recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "huggingface_hub" %}
-{% set version = "0.17.3" %}
+{% set version = "0.20.3" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 40439632b211311f788964602bf8b0d9d6b7a2314fba4e8d67b2ce3ecea0e3fd
+  sha256: 94e7f8e074475fbc67d6a71957b678e1b4a74ff1b64a644fd6cbb83da962d05d
 
 build:
   number: 0
```

Comment:
Remove `importlib-metadata` and add `pydantic >1.1,<3.0  # [py>38` and `pydantic>1.1,<2.0  # [py==38]` to `run_constrained`.

Also change `skip: true  # [py<37]` to `skip: true  # [py<38]`.

Reply:
And modify `fsspec` as `fsspec >=2023.5.0`